src/compositions/WorkshopScreening/LogicBranches.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Question1 from './Question1';
+import Question2 from './Question2';
+import Question3 from './Question3';
+
```

Comment:
For now, this is ok. But this doesn't feel like the logic will scale well to other orgs or for other questionnaires. Will merge but will definitely need to keep an eye on these.